### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by TatjanaOcedova**
Add homework Databases week2

### CODE REVIEW

File: databases/week2/homeworkPart2.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+-- Part 2: School database
+-- Create a new database containing the following tables:
+CREATE DATABASE school
+    DEFAULT CHARACTER SET = 'utf8mb4';
+
+USE school;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE `Class` (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
```
Comment (lawansubba) [inline databases/week2/homeworkPart2.sql:10]: nice :) you created a primary key

File: databases/week2/homeworkPart2.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    `name` varchar(255) NOT NULL,
+    `begins_date` DATETIME NOT NULL,
+    `ends_date` DATETIME NOT NULL
+    );
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE TABLE `Student` (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
```
Comment (lawansubba) [inline databases/week2/homeworkPart2.sql:17]: nice :) you created a primary key

File: databases/week2/homeworkPart3.sql [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+FROM
+    task
+JOIN
+    user
+WHERE
+    user.name = "Maryrose Meadows" AND month(created) = 9;
+
+-- Find how many tasks where created in each month, e.g. how many tasks were created in october, how many tasks were created in november, etc. (hint: use group by)
+SELECT
+    COUNT(task.id) as "created tasks",
```
Comment (lawansubba) [inline databases/week2/homeworkPart3.sql:44]: nice names for columns you have created

**Review by lawansubba — COMMENTED** [review]
From the assignment description at https://github.com/HackYourFuture-CPH/databases/tree/main/lesson2
Part 1, 2, 3 looks good, thank you so much dividing your work into parts.sql

Part 4 appears to be missing

**Comment by TatjanaOcedova** [comment]
Thank you for the review, part 4 is the last file and I don't know why is missing for you, it should be 6 files in total.

File: databases/week2/homeworkPart4.sql [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+SELECT
+    *
+FROM
+    book;
+
+
+-- Forgot to add foreing keys
+ALTER TABLE borrowing_log
+ADD  CONSTRAINT `fk_book` FOREIGN KEY (`book_id`) REFERENCES `book` (`book_id`) ON DELETE CASCADE ON UPDATE CASCADE;
```
Comment (lawansubba) [inline databases/week2/homeworkPart4.sql:44]: looks nice :)

File: databases/week2/homeworkPart4.sql [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+ALTER TABLE borrowing_log
+ADD  CONSTRAINT `fk_member` FOREIGN KEY (`member_id`) REFERENCES `member` (`member_id`) ON DELETE CASCADE ON UPDATE CASCADE;
```
Comment (lawansubba) [inline databases/week2/homeworkPart4.sql:47]: looks nice :)

**Comment by lawansubba** [comment]
Part 4 looks good as well

**Pull request merged**